protocol-units/execution/maptos/dof/src/lib.rs

```diff
@@ -27,7 +27,7 @@ pub trait DynOptFinExecutor {
 	/// Initialize the background task responsible for transaction processing.
 	fn background(
 		&self,
-		transaction_sender: Sender<(u64, SignedTransaction)>,
+		transaction_sender: Sender<Vec<(u64, SignedTransaction)>>,
```

Comment:
I think you can remove all these transaction_sender  / receiver because they shouldn't be used.